scripts/set_gradle_proxy.sh

```diff
@@ -38,6 +38,8 @@ writeProxyStringToGradleProps () {
     fi
 
     local PORT=${PROXY##*:}
+    # Remove trailing slashes
+    PORT=${PORT/%\//}
```

Comment:
One more style nit (Did I say yet we're picky?): We like to group comments with belonging code by adding empty lines before the comment and after the belonging code. Here, however, that would add a lot of empty lines with relation to the length of the code block. So personally, I'd either put the comment after the line like `PORT=${PORT/%\//} # Remove trailing slashes.` or drop it altogether as we also don't have comments in similar code above, and you sufficiently explained it in the commit message. But *if* you keep the comment, please end it with a dot, like a proper sentence written in a book.

Reply:
Also, strictly speaking this would only remove a *single* trailing slash, not multiple ones, so the comment should say `# Remove any single trailing slash.`.

Reply:
You're not wrong, I've removed the comment